lib/utils.pm

```diff
@@ -1385,6 +1386,27 @@ sub get_x11_console_tty {
     return (sle_version_at_least('15')) ? 2 : 7;
 }
 
+sub get_addon_fullname {
+    my ($addon) = @_;
+
+    # extensions product list
+    my %product_list = (
+        'ha'    => 'sle-ha',
+        'geo'   => 'sle-ha-geo',
+        'we'    => 'sle-we',
+        'sdk'   => 'sle-sdk',
+        'live'  => 'sle-live-patching',
+        'asmm'  => 'sle-module-adv-systems-management',
+        'contm' => 'sle-module-containers',
+        'hpcm'  => 'sle-module-hpc',
+        'lgm'   => 'sle-module-legacy',
+        'pcm'   => 'sle-module-public-cloud',
+        'tcm'   => 'sle-module-toolchain',
+        'wsm'   => 'sle-module-web-scripting',
+    );
+    return $product_list{"$addon"};
+}
```

Comment:
lib/utils.pm is already getting quite big so we should look for a better place. And as this is clearly related to SLE only I recommend a class in "lib". E.g. there is "lib/registration" already that seems quite fitting. WDYT?

Reply:
I can't agree with you more, lib/utils is over big. Moved it to lib/registration